plugin/keyring_vault/tests/mtr/is_vault_server_up.inc

```diff
@@ -0,0 +1,54 @@
+if (!$PING_TIMEOUT)
+{
+  --let PING_TIMEOUT=4
+}
+
+if (!$KEYRING_CONF_FILE_1)
+{
+  --die KEYRING_CONF_FILE_1 is not set. Should be pointing to config file.
+}
+
+--let KEYRING_CONF_FILE_1=$KEYRING_CONF_FILE_1
+
+--perl
+  my $input = $ENV{KEYRING_CONF_FILE_1};
+  open(my $IN, "<", $input) or die "$0: Can't open input file $input: $!\n";
+  my $vault_url;
+
+  while (my $row = <$IN>) {
+      if ($row =~ /^vault_url/) {
+	  my ($key, $value) = split(/=/, $row);
+          $value =~ s/.*\/\/(.*):.*/$1/;
+          $vault_url=$value;
+      }
+  }
+
+  # trim both ends
+  $vault_url=~ s/^\s+|\s+$//g;
+
+  use Net::Ping;
+  my $vardir= $ENV{MYSQLTEST_VARDIR} or die "Need MYSQLTEST_VARDIR";
+  my $ping_timeout= $ENV{PING_TIMEOUT} or die "Need PING_TIMEOUT";
+
+  $p = Net::Ping->new();
+  my $ping_successful = 0;
+
+  if ($p->ping("$vault_url", $ping_timeout))
+  {
+    $ping_successful= 1;
+  }
+
+  my $file_name = "$vardir/tmp/ping_result.inc";
+  my $F = IO::File->new($file_name, 'w') or die "Could not open '$file_name' for writing";
+  print $F "--let \$PING_SUCCESSFUL = $ping_successful\n";
+  $F->close();
+  $p->close();
+EOF
+
+--source $MYSQLTEST_VARDIR/tmp/ping_result.inc
+--remove_file $MYSQLTEST_VARDIR/tmp/ping_result.inc
+
+if(!$PING_SUCCESSFUL)
+{
+  --skip Seems that Hashicorp Vault is down.
```

Comment:
... testing server ...

Reply:
I am not sure I understand this comment. Should --skip state something like --skip While testing server turned out that Hashicorp Vault is down ?

Reply:
"Seems that Hashicorp Vault testing server is down"

Reply:
The comment not addressed but too minor to hold the merge on its own